src/tests/asyncTests.ts

```diff
@@ -90,12 +90,12 @@ export class AsyncTests {
       }
 
       const asyncRunResult = await sClient.subscribe(undefined, testRunId);
-      const testRunSummary = await this.checkRunStatus(asyncRunResult.runId);
+      const runResult = await this.checkRunStatus(asyncRunResult.runId);
       return await this.formatAsyncResults(
         asyncRunResult,
         getCurrentTime(),
         codeCoverage,
-        testRunSummary,
+        runResult.testRunSummary,
```

Comment:
by making the change here, testRunSummary will return the result even if the finishedStatuses is Queued and Processing. But what does the result look like?